src/hooks/useAxiosWithAuth0.js

```diff
@@ -13,13 +13,14 @@ THIS useAxiosWithAuth0 HOOK SIMPLY SERVES TO HELP CONNECT THE AXIOS CALL AND THE
 export default function useAxiosWithAuth0() {
   const { getAccessTokenSilently } = useAuth0();
 
-  const axiosWithAuth = token =>
-    axios.create({
+  const axiosWithAuth = token => {
+    return axios.create({
```

Comment:
If I add that console.log({ token }) here I'm still seeing those calls with undefined tokens happening....it doesn't seem to be breaking anything....but it is still happening.

Reply:
Yeah, we're going to have to figure out how to resolve that. Maybe capture it in the issue above? As they are linked closely.